apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -1055,10 +1065,10 @@ internal BigtableClientImpl(BigtableServiceApiClient[] clients, string appProfil
 
         /// <inheritdoc/>
         public override BigtableClient WithAppProfileId(string appProfileId) =>
-            new BigtableClientImpl(_clients, appProfileId);
+            new BigtableClientImpl(_clients, appProfileId, UnderlyingClientSettings);
```

Comment:
I'm not suggesting we change this right now, but there's an interesting oddity that changes to the underlying settings are observed when we call `WithAppProfileId`. It's not quite "the same client but with different app profile ID" due to that. We could clone the settings, if we wanted to.

Reply:
Oh good point. I'll clone them in the constructor.